includes/wikia/services/ArticleVideoService.class.php

```diff
@@ -0,0 +1,119 @@
+<?php
+
+use Swagger\Client\ApiException;
+use Swagger\Client\ArticleVideo\Api\MappingsInternalApi;
+use Swagger\Client\ArticleVideo\Models\Mapping;
+use Wikia\DependencyInjection\Injector;
+use Wikia\Logger\WikiaLogger;
+use Wikia\Service\Swagger\ApiProvider;
+
+/**
+ * Created by PhpStorm.
+ * User: ryba
+ * Date: 11/12/2017
+ * Time: 12:24
+ */
+class ArticleVideoService {
+	const SERVICE_NAME = 'article-video';
+	const INTERNAL_REQUEST_HEADER = 'X-Wikia-Internal-Request';
+
+	private static $articleVideoApiClient = null;
+
+	/**
+	 * @param $cityId
+	 *
+	 * @return Mapping[]
+	 * @internal param $pageId
+	 *
+	 */
+	public static function getFeaturedVideosForWiki( string $cityId ): array {
+		$key = wfMemcKey( 'article-video', 'get-for-product', $cityId );
+
+		return WikiaDataAccess::cache(
+			$key,
+			WikiaResponse::CACHE_SHORT,
```

Comment:
Do we want to give possiblity to purge this cache? For example when adding mapping.

Reply:
Staff users will use this tool, they have access to ?mcache=writeonly